Dockerfile.konflux

```diff
@@ -0,0 +1,72 @@
+
+# Build arguments
+ARG SOURCE_CODE=.
+
+FROM registry.redhat.io/ubi8/go-toolset@sha256:4ec05fd5b355106cc0d990021a05b71bbfb9231e4f5bdc0c5316515edf6a1c96 AS stage
+
+# Define a build argument for the PNC list of built files
+ARG PNC_FILES_JSON
+RUN echo "Files to download: $PNC_FILES_JSON"
+
+# Install packages for the install script and extract archives
+RUN microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install -y unzip jq wget
+
+ENV STAGE_DIR="/tmp/artifacts"
+WORKDIR $STAGE_DIR
+
+# Filter the zip files only and unzip them in /root/
+RUN echo "$PNC_FILES_JSON" | jq -r '.[] | select(test("\\.zip$"))' | \
+    while read url; do wget --no-check-certificate "$url"; done && \
+    for file in *.zip; do unzip -d /root/ "$file"; done
+
+
+###############################################################################
+FROM registry.redhat.io/ubi8/openjdk-17-runtime@sha256:e2f33a6c60db4f4e70882a4a557eec5890997f8a1be7e3eb8971a0ff8a45a1a8 as runtime
+
+## Build args to be used at this step
+ARG USERID=2000
+
+LABEL com.redhat.component="odh-modelmesh-container" \
+      name="managed-open-data-hub/odh-modelmesh-rhel8" \
+      description="Modelmesh is a distributed LRU cache for serving runtime models" \
+      summary="odh-modelmesh" \
+      maintainer="['managed-open-data-hub@redhat.com']" \
+      io.openshift.expose-services="" \
+      io.k8s.display-name="odh-modelmesh" \
+      io.k8s.description="odh-modelmesh" \
+      com.redhat.license_terms="https://www.redhat.com/licenses/Red_Hat_Standard_EULA_20191108.pdf"
+
+USER root
+
+RUN sed -i 's:security.provider.12=SunPKCS11:#security.provider.12=SunPKCS11:g' /usr/lib/jvm/java-17-openjdk-*/conf/security/java.security \
+    && sed -i 's:#security.provider.1=SunPKCS11 ${java.home}/lib/security/nss.cfg:security.provider.12=SunPKCS11 ${java.home}/lib/security/nss.cfg:g' /usr/lib/jvm/java-17-openjdk-*/conf/security/java.security
+
+ENV JAVA_HOME=/usr/lib/jvm/jre-17-openjdk
```

Comment:
the base image already provides this env, IMHO, we don't need to set it here.

Reply:
ok